charts/opentelemetry-collector/README.md

```diff
@@ -192,10 +192,9 @@ presets:
 
 ### Configuration for Annotation-Based Discovery
 
-The collector can be configured to automatically discover and collect telemetry from pods based on annotations. This feature provides a drop-in replacement for the `logsCollection` preset, allowing for selective collection of logs and metrics from specific pods rather than collecting from all pods in the cluster.
+The collector can be configured to automatically discover and collect telemetry from pods based on annotations. This feature provides a drop-in replacement for the `logsCollection` preset, allowing for selective collection of logs.
 
-> [!WARNING]
-> Annotation-based discovery and `logsCollection` are mutually exclusive. You should use either:
+> [!WARNING] > `annotationDiscovery.logs` and `logsCollection` are mutually exclusive. You should use either:
 >
 > - `presets.logsCollection.enabled: true` (collects logs from all pods)
 > - `presets.annotationDiscovery.logs.enabled: true` (collects logs only from pods with specific annotations)
```

Comment:
For some reason I can't recommend changes 🤔 . I post my suggestion here directly:

```
`presets.annotationDiscovery.logs.enabled: true`: Collects logs only from all pods by-default, and allows to define additional configuration through annotations. Log collection from specific Pods/containers, can disabled by using the proper annotation.
```